test-framework/src/test/java/api/app/astrodao/com/tests/subscriprions/SubscriptionsApiTests.java

```diff
@@ -42,10 +43,13 @@ public void cleanUpSubscriptions() {
     void userShouldBeAbleToSubscribeToADao() {
         String dao = "marmaj.sputnikv2.testnet";
         String subscriptionId = String.format("%s-%s", dao, accountId);
-        Response response = subscriptionsApiSteps.subscribeDao(accountId, accountPublicKey, accountSignature, dao);
-        subscriptionsApiSteps.assertResponseStatusCode(response, HttpStatus.CREATED);
 
-        Subscription subscription = subscriptionsApiSteps.getResponseDto(response, Subscription.class);
+        Subscription subscription = subscriptionsApiSteps.subscribeDao(accountId, accountPublicKey, accountSignature, dao).then()
+                .statusCode(HTTP_CREATED)
+                .extract().as(Subscription.class);
+//        subscriptionsApiSteps.assertResponseStatusCode(response, HttpStatus.CREATED);
```

Comment:
I suppose theses commented lines should be removed

Reply:
resolved